test/shared/src/main/scala/zio/test/package.scala

```diff
@@ -84,7 +84,7 @@ package object test {
   /**
    * Builds a suite containing a number of other specs.
    */
-  final def suite[R, E, L](label: L)(specs: ZSpec[R, E, L]*): ZSpec[R, E, L] = Spec.suite(label, specs.toVector, None)
+  final def suite[L, T](label: L)(specs: Spec[L, T]*): Spec[L, T] = Spec.suite(label, specs.toVector, None)
```

Comment:
If you're going to generalize `suite`, I'd do the same for `testM`, which will become identity. 

It's a little strange that `testM` just passes through the test, which is not necessarily monadic, but it will work as expected for most users.

Reply:
Right! Didn't notice that testM was identity through the type aliases D: